src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+import { useSearchParams } from 'react-router-dom';
+import { useState, useEffect, useMemo } from 'react';
+import { PeopleFilters } from '../PeopleFilters';
+import { PeopleTable } from '../PeopleTable';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { getFilteredPeople } from '../../utils/getFilteredPeople';
+import { getSortedPeople } from '../../utils/getSortedPeople';
+
+export const PeoplePage = () => {
+  const [searchParams] = useSearchParams();
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [peopleLoadingError, setPeopleLoadingError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(setPeople)
+      .catch(() => setPeopleLoadingError(true))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  const filteredPeople = useMemo(() => {
+    const filters = {
+      sex: searchParams.get('sex'),
+      name: searchParams.get('query'),
+      centuries: searchParams.getAll('centuries'),
```

Comment:
suggest to make enum for all possible search params.
and use enum instead strings.